notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb

```diff
@@ -4,31 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>Hi @gsanand, the files <code>residuals_{rootname_A}_flc.png"</code>  and  <code>residuals_{rootname_B}_flc.png",</code></p><p>have already been renamed previosly with a trailing<code>_rscale</code> which is why they don't exist. Perhaps, you'll need to change the workflow on this. Thanks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Hi @haticekaratay , the errors I see in the CI are with regards to opening the "_rscale" or other similar files though, and not the ones that end in "_flc" themselves.

for example:

FileNotFoundError: [Errno 2] No such file or directory: 'residuals_j9irw5kaq_flc_rscale.png'

Reply:
Yes, indeed. You are right; I was looking at the previous build logs. Let's get @dulude and @mgough-970's feedback. Hi @dulude and @mgough-970, this notebook (align_multiple_visits) runs successfully locally and produces no errors with local tests:
 `pytest --nbval ./notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb`
  and ` jupyter nbconvert --to html --execute ./notebooks/DrizzlePac/align_multiple_visits/align_multiple_visits.ipynb`. However fails on the validation step in CI.